v2/emailpassword/common-customizations/get-user-info.mdx

```diff
@@ -403,6 +403,42 @@ export default async function likeComment(req: SessionRequest, res: any) {
 }
 ```
 
+</TabItem>
+<TabItem value="nextjs-app">
+
+```tsx
+import { NextResponse, NextRequest } from "next/server";
+import SuperTokens from "supertokens-node";
+import { withSession } from "supertokens-node/nextjs";
+// @ts-ignore
+import { backendConfig } from "@/app/config/backend";
+
+SuperTokens.init(backendConfig());
+
+export function POST(request: NextRequest) {
+    return withSession(request, async (session) => {
```

Comment:
wherever we use withSession, need to surround it with try catch cause see https://github.com/supertokens/supertokens-node/pull/767#discussion_r1435008992.

Reply:
done